### PR TITLE
Add program

### PR DESCRIPTION
**Pull request by RazanTayyem**
- Add program

- [ ]  Create add program page /addProgram in React Router

- [ ]  Create end point  for addProgram Page 

- [ ]  Add Header from Header component

- [ ] Add program form using Form component.

- [ ]  Add Button.

### CODE REVIEW

File: client/src/components/pages/CreateProgram/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import axios from 'axios';
+import Header from "../../Header";
+import MainForm from '../../Form'
+import {Title, StyledPage} from "./index.style";
+
+class CreateProgram extends Component {
+
+  getFieldList() {
+    const fields = [
```
Comment (mhmdtshref) [inline client/src/components/pages/CreateProgram/index.js]: Pun this on the render function before the return.

Comment (RazanTayyem) [reply client/src/components/pages/CreateProgram/index.js]: Done :+1:

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/FACK1/M-A-Platform#readme",
   "dependencies": {
+    "ajv": "^6.8.1",
```
Comment (jema28) [inline package.json]: is this needed for everyone?

Comment (RazanTayyem) [reply package.json]: yes for react

**Pull request merged**